frontend/src/components/pages/AccessManagement/index.tsx

```diff
@@ -0,0 +1,174 @@
+import { SearchIcon } from "@chakra-ui/icons";
+import { FaEllipsisV } from "react-icons/fa";
+import {
+  Container,
+  HStack,
+  IconButton,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Select,
+  Table,
+  Tag,
+  TagLabel,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import React from "react";
+import StatusLabel from "./StatusLabel";
+import UserAPIClient from "../../../APIClients/UserAPIClient";
+import { UserResponse } from "../../../types/UserTypes";
+import { Role } from "../../../types/AuthTypes";
+
+const AccessManagementPage = (): JSX.Element => {
+  enum Filter {
+    ALL = "All",
+    ACTIVE = "Active",
+    INACTIVE = "Inactive",
+  }
+
+  enum UserRoles {
+    ADMIN = "FON Admin",
+    CAMP_COORDINATOR = "Camp Coordinator",
+  }
+
+  const filterOptions = [Filter.ALL, Filter.ACTIVE, Filter.INACTIVE];
+  const userRoles = [UserRoles.ADMIN, UserRoles.CAMP_COORDINATOR];
+
+  const [users, setUsers] = React.useState([] as UserResponse[]);
+  const [search, setSearch] = React.useState("");
+  const [selectedFilter, setSelectedFilter] = React.useState<Filter>(
+    Filter.ALL,
+  );
+
+  React.useEffect(() => {
+    const getUsers = async () => {
+      const res = await UserAPIClient.getAllUsers();
+      console.log(res);
+      if (res.length !== undefined) setUsers(res);
+      // else setDataError(true);
+    };
+
+    getUsers();
+  }, []);
+
+  const tableData = React.useMemo(() => {
+    let filteredUsers;
+    if (selectedFilter === Filter.ALL) filteredUsers = users;
+    else if (selectedFilter === Filter.ACTIVE)
+      filteredUsers = users.filter((user) => user.active);
+    else filteredUsers = users.filter((user) => !user.active);
+
+    if (!search) return filteredUsers;
+    return filteredUsers.filter(
+      (user) =>
+        user.email.toLowerCase().includes(search.toLowerCase()) ||
```

Comment:
The search bar says to search by name; did we want to search by email ID too? If so, I think we should change the search field to say "Search by name or email"